### DISCUSSION TITLE
Score multiplier for DT/NC/HT/DC based on the speed change

### DISCUSSION THREAD

**Discussion by Maarlink** [ppy/osu#19191] [Ideas] [answered]
So I saw this here https://github.com/ppy/osu/discussions/17516#discussioncomment-2456872 as a suggested answer and thought it was a pretty fair idea.

Score multiplier with a speed changing mode should be based on the speed change rather than just a flat score multiplier for the mod.

For exemple, in the screenshots below, we can see that I have the same score with DT x1.1 and with DT x1.5, which seems obviously unfair. 

![osu_2022-07-17_14-59-21](https://user-images.githubusercontent.com/109466041/179399502-bce8142f-7ae7-40d0-a4ed-34814f867108.jpg)
![osu_2022-07-17_14-59-24](https://user-images.githubusercontent.com/109466041/179399505-fe52ee38-189f-48a1-9a01-85051997b433.jpg)
e

**Comment by mk56-spn** [comment]
this is already being debated extensively in the circles that handle the subject.  to my knowledge there is no conclusion as of yet. but it will probably have some variating multiplier as you say

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/7698

Probably should address this and make it worth nothing if adjusted from the default as a first step.
